hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeSchemaTwoDBDefinition.java

```diff
@@ -88,6 +89,10 @@ public DatanodeSchemaTwoDBDefinition(String dbPath,
           DELETED_BLOCKS,
           DELETE_TRANSACTION);
 
+  public static Set<String> getColumnFamiliesName() {
+    return COLUMN_FAMILIES.keySet();
```

Comment:
There is no strong need to add this getColumnFamiliesName function. It can be replaced with getMap().keySet() call.